src/documentation/user_guides/malloy_by_example.malloynb

```diff
@@ -554,8 +523,7 @@ query: q_airport_facts is duckdb.table('../data/flights.parquet') -> {
 
 run: q_airport_facts
 >>>markdown
-
->>>malloy
+\>>>malloy
```

Comment:
Typo?

Reply:
who knows ... notebook editing is scary and weird ... somehow this happened